scripts/ubuntu-bartender/aws-provider

```diff
@@ -25,8 +25,13 @@ function _wait-instance-running() {
 }
 
 function _wait-instance-connectable() {
+  key_opt=''
+  if [ -f ~/.ssh/${AWS_KEYPAIR_NAME}.pem ]; then
+    key_opt="-i ~/.ssh/${AWS_KEYPAIR_NAME}.pem"
+  fi
```

Comment:
can this `if` block live outside of the function definitions? That way, we only define key_opt once?